testing/constraints-3.8.txt

```diff
@@ -1,13 +1,18 @@
 # -*- coding: utf-8 -*-
-# This constraints file is required for unit tests.
+# This constraints file is used to check that lower bounds
+# are correct in setup.py
 # List all library dependencies and extras in this file.
-google-api-core
-libcst
-pandas
-proto-plus
-grpc-google-iam-v1
-google-cloud-bigquery
-google-cloud-documentai
-google-cloud-storage
-numpy==1.21.6
+# Pin the version to the lower bound.
+# e.g., if setup.py has "google-cloud-foo >= 1.14.0, < 2.0.0dev",
+# Then this file should have google-cloud-foo==1.14.0
+google-api-core==2.17.1
+libcst==0.2.5
```

Comment:
Is `libcst` dependency needed? You can automatically generate this file using the steps in https://github.com/googleapis/synthtool/issues/1901#issuecomment-1820742964

Reply:
I had issues when running that tool with this setup.py requirements

```
 install_requires=(
        "google-api-core >= 2.17.1, <3.0.0dev",
        "pandas[performance,gcp] >= 2.0.0, <3.0.0",
        "pyarrow >= 15.0.0, <16.0.0",
        "tabulate >= 0.9.0, <1.0.0",
        "proto-plus >= 1.22.3, <2.0.0dev",
        "grpc-google-iam-v1 >= 0.12.6, < 0.14.0dev",
        "google-cloud-bigquery >= 3.5.0, < 4.0.0dev",
        "google-cloud-documentai >= 2.20.0, < 3.0.0dev",
        "google-cloud-storage >= 1.31.0, < 3.0.0dev",
        "google-cloud-vision >= 2.7.0, < 4.0.0dev",
        "numpy >= 1.23.5, < 2.0.0",
        "intervaltree >= 3.0.0",
        "pikepdf >= 8.0.0, < 9.0.0",
        "immutabledict >= 2.0.0, < 5.0.0",
        "Pillow >= 10.0.0, < 11.0.0",
        "Jinja2 >= 3.1.0, <= 4.0.0",
    ),
```

I keep getting this error:
```
lower-bound-checker update --package-name google-cloud-documentai-toolbox --constraints-file /Users/holtskinner/GitHub/python-documentai-toolbox/testing/constraints-3.8.txt
Usage: lower-bound-checker update [OPTIONS]
Try 'lower-bound-checker update --help' for help.

Error: setup.py is missing explicit lower bounds for the following packages: ['Jinja2', 'google-api-core', 'google-api-core']
```